packages/fuel-indexer-lib/src/utils.rs

```diff
@@ -325,3 +328,11 @@ pub fn host_triple() -> String {
         .expect("Failed to determine host triple via rustc.")[6..]
         .to_owned()
 }
+
+pub fn bool_to_str(b: bool) -> String {
+    if b {
+        "true".to_string()
+    } else {
+        "false".to_string()
+    }
+}
```

Comment:
`bool` implements the `Display` trait, so you should just be able to call `boolean_value.to_string()` instead of having to create a function. Am I missing something here?

Reply:
🤦🏽 brain fart